docs/jwst/refpix/reference_files.rst

```diff
@@ -1,6 +1,22 @@
-Reference File
-==============
+Reference File Types
+====================
+
 The refpix step only uses the refpix reference file when processing
 NIRSpec exposures that have been acquired using an IRS2 readout
 pattern. No other instruments or exposure modes require a reference
 file for this step.
+
+
+CRDS Selection Criteria
+-----------------------
+Refpix reference files are selected by DETECTOR and READPATT.
+
+
+Reference File Format
+---------------------
+
+A single IRS2 extension provides the complex coefficients for the correction,
```

Comment:
Perhaps be a bit more explicit in indicating that the name of the extension, or the value of the EXTNAME keyword, is "IRS2". Also, I notice that the actual ref files in CRDS have a "DQ" extension too. Are those necessary and/or used? If so, they should be documented here too.